appdatabase/migrations/sql/1648333113_ipfs_dir.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE node_config ADD COLUMN ipfs_dir TEXT NOT NULL DEFAULT "";
+UPDATE node_config SET ipfs_dir = "./ipfs";
```

Comment:
@cammellos do you think we could use default path ? or we should set in cljs ?

Reply:
If we don't need to set it, better I suppose, maybe we can get away without storing anything in the database and have a sensible default for both projects?
If we do have to set it, for now we have to do as above, since how we start the node pulls it from the database, but ideally we don't store anything in the db (we can always change that at a later stage, if we have to)

Reply:
I can change it to a default value instead of using the DB. The reason I'm using the node_config tables is so it matches the way we store the datadir and keystore dirs. Do you prefer I use a default path instead?

Reply:
In general, I tend to favor not storing state unless necessary, it's less committal and it's easier to modify etc, but I don't have a strong opinion on the matter, so up to you